docs/ui-ingestion.md

```diff
@@ -3,318 +3,358 @@ import FeatureAvailability from '@site/src/components/FeatureAvailability';
 
-## Running Metadata Ingestion
+This makes it simple to connect to popular platforms like Snowflake, BigQuery, dbt, and more, schedule automatic updates, and manage credentials securely.
 
-### Prerequisites
+## Prerequisites and Permissions
 
-To view & manage UI-based metadata ingestion, you must have the `Manage Metadata Ingestion` & `Manage Secrets`
-privileges assigned to your account. These can be granted by a [Platform Policy](authorization/policies.md).
+To manage metadata ingestion in DataHub, you need appropriate permissions.
 
-<p align="center">
-  <img width="70%"  src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/ingestion-privileges.png"/>
-</p>
+### Option 1: Admin-Level Access
+
+Users can be granted the following privileges for full administrative access to all ingestion sources:
 
-Once you have these privileges, you can begin to manage ingestion by navigating to the 'Ingestion' tab in DataHub.
+- **`Manage Metadata Ingestion`** - Provides complete access to create, edit, run, and delete all ingestion sources
+- **`Manage Secrets`** - Allows creation and management of encrypted credentials used in ingestion configurations
+
+These privileges can be granted in two ways:
+
+1. **Admin Role Assignment** - Users assigned to the **Admin Role** receive these privileges by default
+2. **Custom Policy with Platform Privileges** - Create a [Custom Policy](authorization/policies.md) that grants the `Manage Metadata Ingestion` and `Manage Secrets` platform privileges to specific users or groups
 
 <p align="center">
-  <img width="70%"  src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/ingestion-tab.png"/>
+  <img width="80%"  src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/ingestion-privileges.png"/>
 </p>
 
-On this page, you'll see a list of active **Ingestion Sources**. An Ingestion Sources is a unique source of metadata ingested
-into DataHub from an external source like Snowflake, Redshift, or BigQuery.
+### Option 2: Resource-Specific Policies
 
-If you're just getting started, you won't have any sources. In the following sections, we'll describe how to create
-your first **Ingestion Source**.
+For more granular control, administrators can create [Custom Policies](authorization/policies.md) that apply specifically to **Ingestion Sources**, allowing different users to have different levels of access:
 
-### Creating an Ingestion Source
+- **View** - View ingestion source configurations and run history
+- **Edit** - Modify ingestion source configurations
+- **Delete** - Remove ingestion sources
+- **Execute** - Run ingestion sources on-demand
 
-<Tabs>
-  <TabItem value="ui" label="UI" default>
+### Accessing the Ingestion Interface
 
-Before ingesting any metadata, you need to create a new Ingestion Source. Start by clicking **+ Create new source**.
+Once you have the appropriate privileges, navigate to the **Ingestion** tab in DataHub.
```

Comment:
so in order to get to the ingestion page without manage privileges on ingestion sources, you will need to enable the new `VIEW_INGESTION_SOURCE_PRIVILEGES_ENABLED` flag. This is to prevent accidentally enabling everyone to view the managed ingestion page due to our default "everyone can view everything" policy